llvm/lib/Target/AIE/AIE2TargetMachine.cpp

```diff
@@ -51,6 +51,11 @@ static cl::opt<bool>
     EnableSuperRegSplitting("aie-split-superregs", cl::Hidden, cl::init(true),
                             cl::desc("Enable splitting super-regs into their "
                                      "smaller components to facilitate RA"));
+
+static cl::opt<bool> EnableWAWRegRewrite(
+    "aie-wawreg-rewrite",
+    cl::desc("Enable the WAW Register Renaming in loops (default true)"),
```

Comment:
Nit: drop ` (default true)`, I think it will be printed in the help message because you define a default

Reply:
It will not be printed in the help message, but no other help message contains the default, so i will harmonize it.